src/wp-includes/pluggable.php

```diff
@@ -2448,16 +2448,23 @@ function wp_salt( $scheme = 'auth' ) {
 			 * https://i18n.svn.wordpress.org/<locale code>/branches/<wp version>/dist/wp-config-sample.php
 			 */
 			$duplicated_keys[ __( 'put your unique phrase here' ) ] = true;
+			$option_prime = array();
 
 			foreach ( array( 'AUTH', 'SECURE_AUTH', 'LOGGED_IN', 'NONCE', 'SECRET' ) as $first ) {
 				foreach ( array( 'KEY', 'SALT' ) as $second ) {
 					if ( ! defined( "{$first}_{$second}" ) ) {
+						if ( 'SECRET' !== $first ) {
+							$option_prime[] = strtolower( "{$first}_{$second}" );
+						}
 						continue;
 					}
 					$value                     = constant( "{$first}_{$second}" );
 					$duplicated_keys[ $value ] = isset( $duplicated_keys[ $value ] );
```

Comment:
This accounts for duplicate values and constants defined with the value from the sample config file, ie `define( 'thing', 'put your unique phrase here' )`.

```suggestion
					$duplicated_keys[ $value ] = isset( $duplicated_keys[ $value ] );
					if ( true === $duplicated_keys[ $value ] ) {
						$option_prime[] = strtolower( "{$first}_{$second}" );
					}
```